.github/workflows/tests.yaml

```
@@ -59,15 +31,69 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install Base Dependencies
         run: |
+          echo $COVERAGE_FILE
           python -m pip install --upgrade pip setuptools wheel
           pip install --upgrade tox
-      - name: Run Tests (Tox)
-        run: tox -e py
-      - uses: 5monkeys/cobertura-action@master
-        name: Upload Coverage Report
+      - name: Run ${{ matrix.tests }} Tests
+        run: |
+          echo $COVERAGE_FILE
+          tox -e ${{ matrix.tests }}
+          ls -al
+          ls -al .tox
+      - uses: actions/upload-artifact@v2
         if: ${{ matrix.python-version == 3.9}}
         with:
-          path: coverage.xml
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          minimum_coverage: 75
-          fail_below_threshold: false
+          name: coverage {{ matrix.tests }}
+          path: ${{ env.COVERAGE_FILE }}
+
+  # unit-tests:
+  #   name: Python ${{ matrix.python-version }} - Unit
+  #   runs-on: ubuntu-latest
+  #   strategy:
+  #     matrix:
+  #       python-version: [3.8, 3.9]
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #       with:
+  #         lfs: true
+  #     - name: Set up Python ${{ matrix.python-version }}
+  #       uses: actions/setup-python@v2
+  #       with:
+  #         python-version: ${{ matrix.python-version }}
+  #     - name: Install Base Dependencies
+  #       run: |
+  #         python -m pip install --upgrade pip setuptools wheel
+  #         pip install --upgrade tox
+  #     - name: Run Tests
+  #       run: |
+  #         tox -e unit
+  #         ls -al
+  #     - uses: actions/upload-artifact@v2
+  #       if: ${{ matrix.python-version == 3.9}}
+  #       with:
+  #         name: unit-tests
+  #         path: ${{ env.COVERAGE_FILE }}
+
+  upload-to-codecov:
+    name: Upload Coverage to Codecov
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
```

Trengs noe sånt for å laste ned riktig artifact? 
```suggestion
        uses: actions/download-artifact@v2
        with:
          name: coverage*
```
https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts#passing-data-between-jobs-in-a-workflow

---

I believe that it downloads everything without a name.  But feel free to test it out.  Not sure why it won’t work.

---

Ok. I will push a test.

---

I fixed some with how artifacts are uploaded, and it is now passing. Do you see in the codecov report that it looks ok?

---

I were able to see the result now from link in checks, not following the link in comment above. Looks to be working.